qadi_simple_multi.py

```
@@ -23,62 +23,138 @@
 
 sys.path.insert(0, str(Path(__file__).parent / 'src'))
 
-async def run_qadi_phase(phase_name: str, prompt: str, previous_insights: str = ""):
-    """Run a single QADI phase using simple LLM call."""
+async def run_qadi_phase(phase_name: str, prompt: str, previous_insights: str = "", concrete_mode: bool = False, classification_result=None):
+    """Run a single QADI phase using adaptive prompts based on question classification."""
     from mad_spark_alt.core.llm_provider import setup_llm_providers, llm_manager, LLMRequest
```

Ruff (0.12.2)

`mad_spark_alt.core.llm_provider.setup_llm_providers` imported but unused

Remove unused import: `mad_spark_alt.core.llm_provider.setup_llm_providers`

(F401)